.github/scripts/clone_pg_hint_plan_repo

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+
+set -e
+
+BASE_URL=https://github.com/ossc-db/pg_hint_plan
+GIT_URL=${BASE_URL}.git
+
+echo running git clone "$GIT_URL"
+git clone "$GIT_URL" && exit
```

Comment:
Do we really want to pull their master (unreleased) branch? I think at minimum we'd also want to pull the latest release of pg_hint_plan.

Reply:
I've added a new configuration file from which the release version of pg_hint_plan is picked up